autogpt_platform/backend/schema.prisma

```diff
@@ -19,45 +19,164 @@ model User {
   updatedAt DateTime @updatedAt
 
   // Relations
-  AgentGraphs                  AgentGraph[]
+  AgentGraphs                  Agent[]
   AgentGraphExecutions         AgentGraphExecution[]
   AgentGraphExecutionSchedules AgentGraphExecutionSchedule[]
   AnalyticsDetails             AnalyticsDetails[]
   AnalyticsMetrics             AnalyticsMetrics[]
   UserBlockCredit              UserBlockCredit[]
+  ConfiguredAgents             AgentPreset[]
+  UserAgents                   UserAgents[]
+
+  // User Group relations
+  UserGroupMemberships   UserGroupMembership[]
+  UserAccounting         UserAccounting[]
+  UserSubscription       UserSubscription[]
+  Profile                Profile[]
+  StoreListing           StoreListing[]
+  StoreListingSubmission StoreListingSubmission[]
+}
+
+model UserGroup {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  name         String
+  description  String
+  groupIconUrl String?
+
+  UserGroupMemberships UserGroupMembership[]
+
+  Agents       Agent[]
+  Profile      Profile[]
+  StoreListing StoreListing[]
+
+  @@index([name])
+}
+
+enum UserGroupRole {
+  MEMBER
+  OWNER
+}
+
+model UserGroupMembership {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  userId      String
+  user        User          @relation(fields: [userId], references: [id])
+  userGroupId String
+  userGroup   UserGroup     @relation(fields: [userGroupId], references: [id])
```

Comment:
relations should be PascalCase, also in the rest of the file

Reply:
Hopefully got them all now